solid-crdt.ttl

```diff
@@ -0,0 +1,120 @@
+@prefix : <https://w3.org/ns/crdt#> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix dcmi: <http://purl.org/dc/dcmitype/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#dateTime> .
+
+<https://w3.org/ns/crdt#>
+    a owl:Ontology ;
+    rdfs:label "Conflict-Free Replicated Datatypes (CRDT)"@en ;
+    dc:description "The Conflict-Free Replicated Datatypes (CRDT) vocabulary can be used to describe changes made to resources over time"@en ;
+    rdfs:isDefinedBy : ;
+    dc:issued "2022-05-26"^^xsd:date ;
+    vann:preferredNamespacePrefix "crdt"@en ;
+    vann:preferredNamespaceUri "http://www.w3.org/ns/solid/crdt#"^^xsd:anyURI .
+
+:Metadata
+    a rdfs:Class ;
+    rdfs:label "Metadata"@en ;
+    dc:description "Metadata about a resource."@en ;
+    rdfs:subClassOf dcmi:Dataset ;
+    rdfs:isDefinedBy : .
+
+:Operation
+    a rdfs:Class ;
+    rdfs:label "Operation"@en ;
+    dc:description "Operation performed on a resource."@en ;
+    rdfs:subClassOf dcmi:Dataset ;
+    rdfs:isDefinedBy : .
+
+:OperationType
+    a rdf:Class ;
+    rdfs:label "Operation Type"@en ;
+    dc:description "Type of the operation."@en ;
+    rdfs:isDefinedBy : .
+
+:resource
+    a rdf:Property ;
+    rdfs:label "resource"@en ;
+    dc:description "If the object is a :Metadata, resource which the metadata makes reference to. If the object is a :Operation, resource affected by the operation."@en ;
+    rdfs:subPropertyOf dcmi:subject ;
+    rdfs:range rdfs:Resource ;
+    rdfs:domain :Metadata, :Operation ;
+    rdfs:isDefinedBy : .
+
+:createdAt
+    a rdf:Property ;
+    rdfs:label "createdAt"@en ;
+    dc:description "Time at which the resource was created."@en ;
+    rdfs:subPropertyOf dcmi:created ;
+    rdfs:range xsd:dateTime ;
+    rdfs:domain :Metadata ;
+    rdfs:isDefinedBy : .
+
+:updatedAt
+    a rdf:Property ;
+    rdfs:label "updatedAt"@en ;
+    dc:description "Time at which the resource was updated for the last time."@en ;
+    rdfs:subPropertyOf dcmi:modified ;
+    rdfs:range xsd:dateTime ;
+    rdfs:domain :Metadata ;
+    rdfs:isDefinedBy : .
+
+:type
+    a rdf:Property ;
+    rdfs:label "type"@en ;
+    dc:description "Type of the operation. If this property is missing, :SetOperation will be assumed."@en ;
```

Comment:
In general defaults like this have the problem that RDF is monotonic in that any subset of a true graph is also true, so the graph without the property can be inferred from the graph with the property.

Reply:
Thanks for pointing this out. I think I understand what you mean, but in that case how could this be modeled? I've looked at `schema.org`, and they have some properties that as far as I understand have defaults. For example, the [Offer](https://schema.org/Offer) class has a [businessFunction](https://schema.org/businessFunction) property and they say that the default is [Sell](http://purl.org/goodrelations/v1#Sell).